docs/api/toast.md

```diff
@@ -22,78 +22,29 @@ import EncapsulationPill from '@components/page/api/EncapsulationPill';
 
 A Toast is a subtle notification commonly used in modern applications. It can be used to provide feedback about an operation or to display a system message. The toast appears on top of the app's content, and can be dismissed by the app to resume user interaction with the app.
 
-## Presenting
+## Inline Toasts (Recommended)
 
-### Positioning
+`ion-toast` can be used by writing the component directly in your template. This reduces the number of handlers you need to wire up in order to present the toast.
 
-Toasts can be positioned at the top, bottom or middle of the viewport. The position can be passed upon creation. The possible values are `top`, `bottom` and `middle`. If the position is not specified, the toast will be displayed at the bottom of the viewport.
-
-### Controller
+import InlineToastTriggerExample from '@site/static/usage/v7/toast/inline/basic/index.md';
 
-import ControllerExample from '@site/static/usage/v7/toast/presenting/controller/index.md';
+<InlineToastTriggerExample />
 
-<ControllerExample />
+### Using `isOpen​`
 
-### Inline
+The `isOpen` property on `ion-toast` allows developers to control the presentation state of the toast from their application state. This means when `isOpen` is set to `true` the toast will be presented and when `isOpen` is set to `false` the toast will be dismissed.
 
-When using Ionic with React or Vue, `ion-toast` can also be placed directly in the template through use of the `isOpen` property. Note that `isOpen` must be set to `false` manually when the toast is dismissed; it will not be updated automatically.
+`isOpen` uses a one-way data binding, meaning it will not automatically be set to `false` when the toast is dismissed. Developers should listen for the `ionToastDidDismiss` or `didDismiss` event and set `isOpen` to `false`. The reason for this is it prevents the internals of `ion-toast` from being tightly coupled with the state of the application. With a one way data binding, the toast only needs to concern itself with the boolean value that the reactive variable provides. With a two way data binding, the toast needs to concern itself with both the boolean value as well as the existence of the reactive variable itself. This can lead to non-deterministic behaviors and make applications harder to debug.
 
-<Tabs defaultValue="react" values={[{ value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
-<TabItem value="react">
+import InlineToastIsOpenExample from '@site/static/usage/v7/toast/inline/is-open/index.md';
 
-```tsx
-import React, { useState } from 'react';
-import { IonButton, IonToast } from '@ionic/react';
+<InlineToastIsOpenExample />
```

Comment:
Optional: Should we have a `duration` set on this toast, even if it's a really long one? We generally encourage this as a best practice, so it might be good to demonstrate here as well.

Reply:
Yep good point - I'll add it

Reply:
Fixed in https://github.com/ionic-team/ionic-docs/pull/2677/commits/c7256097600716cfd7ec39ce6d30fd05778c62f7

I ended up removing the buttons altogether and had the example show how to update the state of `isOpen` when the toast dismisses